modules/monitoring-managing-user-permissions-using-web-ui.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * monitoring/monitoring-your-own-services.adoc
+
+[id="managing-user-permissions-using-web-ui_{context}"]
+= Managing user permissions using web UI
+
+This procedure shows how to manage user permissions for monitoring their own services using the web UI.
+
+.Prerequisites
+
+* You need to have a user created.
+* Log in the web console as a cluster administrator.
+
+.Procedure
+
+This example shows how to assign permissions to user `johnsmith` for setting up metrics collection and creating alerting rules in the `ns1` namespace:
```

Comment:
I've been told that intro sentences should not follow the Procedure subhead. In this case, it is confusing because in the intro sentence to this module, it says that the proc shows how to manage user perms, and here it says it shows how to assign them. They seem to both mean the same thing. I realize you are using example values, so I'm wondering if you can use annotations as I suggested in the CLI section.

Reply:
Changed accordingly.